client/core/core.go

```diff
@@ -1438,7 +1456,11 @@ func (c *Core) Withdraw(pw []byte, assetID uint32, value uint64) (asset.Coin, er
 	if !found {
 		return nil, fmt.Errorf("%s wallet not found", unbip(assetID))
 	}
-	coin, err := wallet.Withdraw(wallet.address, value, wallet.Info().DefaultFeeRate)
```

Comment:
`wallet.address` being the core of the error here I suppose?

Reply:
Yeah. Wallet was sending to itself.